app/models/miq_report.rb

```diff
@@ -271,7 +271,7 @@ def validate_sorting_columns(columns)
 
   def validate_columns(sorting_columns)
     Array(sorting_columns).collect do |attr|
-      if col_order&.include?(attr)
+      if cols&.include?(attr)
```

Comment:
`col_order` has columns that `cols` does not. So if anything, we would want both.
```suggestion
      if col_order&.include?(attr) || cols&.include?(attr)
```